css/main.css

```diff
@@ -0,0 +1,159 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+    box-sizing: border-box;
+}
+
+.container{
+    width: 100%;
+    height: 100vh;
+    background-image: url(../img/image\ 141.png);
+    background-size: cover;
+    background-position: center;
+    position: absolute;
+}
+
+.header{
+    width: 1206px;
+    height: 80px;
+    background: #F3F4F6;
+    margin: 24px auto;  
+}
+
+nav{
+    display: flex;
+    align-items: center; 
+    /* justify-content: space-between; */
+}
+
+.logo{
+ width: 116px;
+ height: 26px;
+ margin-top: 27px;
+ margin-left: 32px;
+ margin-right: 71px;
+}
+
+.link li{
+    display: inline-block;
+    list-style: none;
+    margin-right: 40px;
+    margin-top: 32px;
+    
+    
+}
+
+a{
+    text-decoration: none;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 18.75px;
+    color: black;
+}
+
+a::after{
+    content: "";
+    width: 0%;
+    height: 2px;
+    background-color: #f44336;
+    display: block;
+    margin: auto;
+    transition: 0.5s;
+}
+
+a:hover::after{
+width: 100%;
+}
+
+.first_icon{
+ margin-left: 400px;
+}
+
+.icon{
+    margin-right: 20px;
+    cursor: pointer;
+}
+
+
+
+h1{
+    font-size: 52px;
+    font-weight: bold;
+    line-height: 61px;
+    text-align: center;
+    margin-top: 96px;
+    color: white;
+}
+
+p{
+    font-size: 18px;
+    line-height: 21px;
+    text-align: center;
+    margin-top: 22px;
+    color: white;
+}
+
+.button{
+    width: 200px;
+    height: 50px;
+    background: #fff;
+    border: 1px solid #000000;
+    box-sizing: border-box;
+    border-radius: 5px;
+    font-size: 16px;
+line-height: 19px;
+margin-left: 684.5px;
```

Comment:
bad idea to centre button, because I have a smaller screen, and button isn't centred for me
do it in another way
for example, with `position: relative, left and transform`